src/main/java/io/xpring/xrpl/Utils.java

```diff
@@ -116,4 +120,96 @@ public static boolean isValidClassicAddress(String address) {
   public static String toTransactionHash(String transactionBlobHex) {
     return javaScriptUtils.toTransactionHash(transactionBlobHex);
   }
+
+  /**
+   * Convert from units in drops to XRP.
+   *
+   * @param drops An amount of XRP expressed in units of drops.
+   * @return A String representing the drops amount in units of XRP.
+   * @throws IllegalArgumentException if drops is in an invalid format.
+   */
+  public static String dropsToXrp(String drops) throws IllegalArgumentException {
+    Preconditions.checkNotNull(drops);
+
+    String dropsRegex = "^-?[0-9]*['.']?[0-9]*$";
+    Pattern dropsPattern = Pattern.compile(dropsRegex);
+    Matcher dropsMatcher = dropsPattern.matcher(drops);
+
+    if (!dropsMatcher.matches()) {
+      throw new IllegalArgumentException(String.format(
+              "dropsToXrp: invalid value %s, should be a string-encoded number matching %s.", drops, dropsRegex));
+    } else if (drops.equals(".")) {
+      throw new IllegalArgumentException(String.format(
+              "dropsToXrp: invalid value %s, should be a string-encoded number.", drops));
+    }
+
+    // Converting with toBigIntegerExact() will throw an ArithmeticException if there is a fractional remainder.
+    // Drops values can be expressed as a decimal (i.e. 2.00) but still must be whole numbers.
+    // Important: specify base 10 to avoid exponential notation, e.g. '1e-7'
+    try {
+      drops = new BigDecimal(drops).toBigIntegerExact().toString(10);
+    } catch (ArithmeticException exception) {     // drops are only whole units
+      throw new IllegalArgumentException(String.format("dropsToXrp: value %s must be a whole number.", drops));
+    }
+
+    // This should never happen; the value has already been validated above.
+    // This just ensures BigDecimal did not do something unexpected.
+    if (!dropsMatcher.matches()) {
+      throw new IllegalArgumentException(String.format(
+              "dropsToXrp: failed sanity check - value %s does not match %s.", drops, dropsRegex));
+    }
+
+    return new BigDecimal(drops).divide(new BigDecimal(1000000.0)).toPlainString();
```

Comment:
You use this value twice. Would it make it more clear to name it (precision?) or calculate so the reader understands where it's coming from:

scale = 8
precision = Math.pow(10, scale)

Reply:
I used a named variable instead of the magic number 1000000.0 to make it more clear.